hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/util/TestHiveSchemaUtil.java

```diff
@@ -145,4 +145,11 @@ public void testSchemaDiffForTimestampMicros() {
         schemaDifference.getAddColumnTypes(), Collections.emptyList(), true);
     assertTrue(schemaDifference.isEmpty());
   }
+
+  @Test
+  public void testGenerateCreateDataBaseDDL() {
+    String expectedCreateDataBaseSQL = "CREATE DATABASE IF NOT EXISTS `test_database`";
+    String testDataBase = HiveSchemaUtil.generateCreateDataBaseDDL("test_database");
+    assertEquals(expectedCreateDataBaseSQL, testDataBase);
+  }
```

Comment:
Can we also have a test case for create table?

Reply:
Thanks for your suggestion, I will submit the test code as soon as possible.